src/TestcafeGherkinBootstrapper.js

```diff
@@ -132,4 +138,22 @@ module.exports = class TestcafeGherkinBootstrapper extends TestcafeBootstrapper
       await this._runStep(hook.code, testController, []);
     }
   }
+
+  _shouldRunScenario(scenario) {
+    if (!this.tags || !this.tags.length) {
```

Comment:
How can `'this.tags` be falsy?

Reply:
I guess it cannot be. I'll verify and remove it.